Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM haskell:8.4
+
+VOLUME /repo
+WORKDIR /app
+
+RUN stack update
+
+COPY toodles.cabal /app
+COPY stack.yaml /app
+
+RUN stack install --only-dependencies
+
+COPY . /app
+
+RUN stack install
+
+EXPOSE 9001
+
+CMD ["toodles","-d","/repo/"]
+
+# TODO - use a multi stage build to reduce the final image size
```

Comment:
Totally optional - but might be good for us to have `Dockerfile` in the list of scanned files, so we can see this in toodles itself ;)

Reply:
Though if you'd prefer to leave this one out for this PR that's totally fine, up to you

Reply:
Unfortunately I am absolutely unfamiliar with Haskell. Looking at https://github.com/aviaviavi/toodles/blob/master/app/Main.hs#L301 this seems to be a matter of just adding the file type and the comment style to an array type of list, but all other entries are file extensions, and for the Dockerfile, its really just the name.

Reply:
Right, it may work by just adding Dockerfile as is, but I'm not sure off the top of my head. Ok let's skip that for now then. I'll do some more testing of this change myself tonight and will get back to you on merging. Thanks!

Reply:
Yes, just adding it does not seem to do the trick (I tried).